source/renderer/app/stores/VotingStore.js

```diff
@@ -433,8 +447,19 @@ export default class VotingStore extends Store {
     }
   };
 
-  @action _checkVotingRegistrationEnd = () => {
-    this.isRegistrationEnded = new Date() >= VOTING_REGISTRATION_END_DATE;
+  @action _checkFundPhase = (now: Date) => {
+    const phaseValidation = {
+      [FundPhases.SNAPSHOT]: now < VOTING_CAST_START_DATE,
+      [FundPhases.VOTING]:
+        now >= VOTING_CAST_START_DATE && now <= VOTING_CAST_END_DATE,
+      [FundPhases.TALLYING]:
+        now > VOTING_CAST_END_DATE && now < VOTING_RESULTS_DATE,
+      [FundPhases.RESULTS]: now >= VOTING_RESULTS_DATE,
+    };
+    this.fundPhase =
+      Object.keys(FundPhases)
+        .map((phase) => FundPhases[phase])
```

Comment:
I had to use this:
`Object.keys(FundPhases).map((phase) => FundPhases[phase])`
Instead of:
`Object.values(FundPhases)`
Due to flow not inferring the type correctly


https://github.com/facebook/flow/issues/2221